camelot/view/action_steps/change_object.py

```diff
@@ -536,18 +537,22 @@ class ChangeField( ActionStep ):
 
     """
 
-    admin: ApplicationAdmin
+    admin: InitVar[ApplicationAdmin]
     field_attributes: InitVar = None
     field_name: str = None
     field_value: str = None
```

Comment:
dat dit een str is vind ik wel verdacht, not sure of deze step wel gaat werken na serialisatie.

ben ook nt zeker of dit hier nog wel een step moet zijn ... en we dit nt beter op een andere wijze doen ...

Reply:
Ok, zie 96858fb3afb15ec32ac479f2e38f7eadf1f8494d